### PR TITLE
changes for about us section

### PR DESCRIPTION
**Pull request by Sarthak-Mittal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,7 +82,7 @@
 
                         <h1 class="brand-heading">Breeze '16</h1>
                         <p class="intro-text">Shiv Nadar University</p>
-                        <a href="#contact" class="btn btn-circle page-scroll">
```
Comment (FlameFractal) [inline index.html:82]: Should've left the link for Contact section too, and added another for About Us.
Contact section could have the link to our youtube, facebook and instagram pages.

Comment (Sarthak-Mittal) [inline index.html:82]: Scroll btn link for Contact section is already made at the end of About Us section. The btn you pointed out is only to reach the next immediate section wiz. About Us.

Comment (FlameFractal) [inline index.html:82]: Oh sorry didn't see that

**Comment by ghost** [comment]
@Sarthak-Mittal @FlameFractal  , I have made a small change in the css file for all section heading tags. Can you have a look at them and edit your heading tag as required ? If it doesn't look good, well, I can revert the changes...The contact section now looks like this #10 

Thanks !

**Comment by ghost** [comment]
And @Sarthak-Mittal , Can you please merge the changes with your own gh-pages branch, so that we can see the changes visually. Doing that renders the website on username.github.io/breeze-website/, just like it did for me - https://harshitbudhraja.github.io/breeze-website/

**Comment by Sarthak-Mittal** [comment]
@harshitbudhraja , do you mind telling me how to do that. I am not so comfortable with GitHub.

**Comment by rhnvrm** [comment]
@Sarthak-Mittal LGTM :+1: 

Merged!

Although the content team has given us too many words, i'll ask them if we an cut it short. If they say we have to use the whole paragraph, we will reduce the font size later on.

**Pull request merged**